### DISCUSSION TITLE
Welcome to topframe Discussions!

### DISCUSSION THREAD

**Discussion by progrium** [progrium/topframe#1] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by deuxlames** [comment]
Hello, i'm just dropping a line to say i'm very interested in the future development of your projet. I'm longing to see a windows-10 port as it's what 'im running. I can't really help at the moment from both lack of time and lack of Golang knowledge, but i'll happily test anyway !
Take care.

**Comment by programmingkidx** [comment]
I have thought about porting MacDriver to other other platforms as well. If MacDriver is to run on Windows 10 then it would probably use [GNUStep](https://gnustep.github.io).

**Reply by progrium** [reply]
I think that's the wrong place to do it. GNUStep is cool, but has its issues. My intention was to port topframe to Apptron for cross platform APIs.

**Reply by programmingkidx** [reply]
This is what you mean: https://github.com/tractordev/apptron

**Comment by progrium** [comment]
correct

On Sun, Jul 16, 2023 at 8:07 PM programmingkidx ***@***.***>
wrote:

> This is what you mean: https://github.com/tractordev/apptron
>
> —
> Reply to this email directly, view it on GitHub
> <https://github.com/progrium/topframe/discussions/1#discussioncomment-6463128>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/AAAAFB5MTUYPQR33MJ3MJGTXQSF5BANCNFSM4ZEP5EDA>
> .
> You are receiving this because you authored the thread.Message ID:
> ***@***.***>
>
-- 
Jeff Lindsay
*https://progrium.xyz <https://progrium.xyz>*
